draft-irtf-cfrg-aegis-aead.md

```diff
@@ -303,13 +303,19 @@ The document is a product of the Crypto Forum Research Group (CFRG). It is not a
 
 # Introduction
 
-This document describes the AEGIS family of authenticated encryption with associated data (AEAD) algorithms {{AEGIS}}, which were chosen as additional finalists for high-performance applications in the Competition for Authenticated Encryption: Security, Applicability, and Robustness (CAESAR). Whilst AEGIS-128 was selected as a winner for this use case, AEGIS-128L has a better security margin alongside improved performance and AEGIS-256 uses a 256-bit key {{LIMS21}}. All variants of AEGIS are inverse-free and constructed from the AES encryption round function {{!FIPS-AES=FIPS.197.2001}}. This document specifies:
+This document describes the AEGIS family of authenticated encryption with associated data (AEAD) algorithms {{AEGIS}}, which were chosen as additional finalists for high-performance applications in the Competition for Authenticated Encryption: Security, Applicability, and Robustness (CAESAR).
+
+All variants of AEGIS are inverse-free and constructed from the AES encryption round function {{!FIPS-AES=FIPS.197.2001}}.
```

Comment:
Should we move this under the bullet points?

Reply:
That would be better, but then it gets a bit redundant:

    This document describes...
    This document specifies:

Maybe we could move it after "Whilst AEGIS-128 was selected as a winner for this use case", rather than right after the bullet list?

Reply:
> That would be better, but then it gets a bit redundant:

Could we change it to `The variants are as follows:` or similar?

> Maybe we could move it after "Whilst AEGIS-128 was selected as a winner for this use case", rather than right after the bullet list?

If you think that would be better. An alternative would be with the above rewording, you could also reword `All variants of AEGIS` to `These are all inverse-free and constructed from the AES encryption round function`. I think it'll work either way around.

Reply:
What do you think about the current version?

Reply:
That's good. I'd probably change `accordingly` to `therefore` because some of the previous sentences started with words beginning with a. Up to you though. I'm just being nitpicky.